Jenkinsfile

```diff
@@ -21,6 +21,7 @@ pipeline {
 						-Pbree-libs \
 						-Papi-check \
 						-Pjavadoc \
+						-Peclipse-sign \
```

Comment:
I think this should only be done on `MASTER` branch!

Reply:
If you change the multi-line string with leading/trailing three single-quotes to an interpolated GroovyString with three double-quotes you could probably something like.
```suggestion
						${env.BRANCH_NAME=='master' ? '-Peclipse-sign': ''} \
```

Reply:
Done